WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/RemoteOrderSynchronizer.swift

```diff
@@ -147,6 +149,27 @@ private extension RemoteOrderSynchronizer {
             }
             .store(in: &subscriptions)
 
+        setProducts.withLatestFrom(orderPublisher)
```

Comment:
I see that this code is pretty similar to what we have in `setProduct.withLatestFrom(orderPublisher)`. Can we reuse the code?

Reply:
I think at least this one should be done here, as we are merging exact duplicated code into `trunk`.

Reply:
Thanks for taking the time to take a look into this with me! Just dropping a link to the [comment here](https://github.com/woocommerce/woocommerce-ios/pull/9173#discussion_r1140045182) so doesn't seem like I skipped this one :D